# Control topic fed into mechanical interface

uint8 control_type	#0 - motion only, 1 - actuator only, 2 - both
geometry_msgs/Twist motion	#2D simple motion
